error.md

```
@@ -0,0 +1,312 @@
+[21:30:26.876] Running build in Washington, D.C., USA (East) – iad1
+[21:30:26.877] Build machine configuration: 2 cores, 8 GB
+[21:30:26.891] Cloning github.com/Yoshinoya-1618/MIXAI (Branch: master, Commit: b26ad84)
+[21:30:26.899] Skipping build cache, deployment was triggered without cache.
+[21:30:27.558] Cloning completed: 667.000ms
+[21:30:27.925] Running "vercel build"
+[21:30:28.381] Vercel CLI 47.0.5
+[21:30:28.733] Warning: Detected "engines": { "node": ">=20" } in your `package.json` that will automatically upgrade when a new major Node.js Version is released. Learn More: http://vercel.link/node-version
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)